copyright

```diff
@@ -1062,6 +1062,11 @@ Copyright: Benjamin Jackson (gods.benyamin@outlook.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license)
 
+Files:
+ image/effect/firestorm?ring*
+License: N/A (CC0 Public Domain)
```

Comment:
That's not a valid license btw. Is it just public domain?

Reply:
Nomadic created it but he labeled all his work under public domain

Reply:
How would you change that?

Reply:
![image](https://user-images.githubusercontent.com/17688683/174737586-71357c78-b829-4763-9651-5fef6e6fbb05.png)

Reply:
I found some examples of it used in older copyright additions and used those.